src/commands/targeting/get.ts

```diff
@@ -42,7 +42,7 @@ export default class DetailedTargeting extends Base {
                     token: this.authToken,
                     projectKey: this.projectKey
                 }
-            )
+            ) as Record<string, any>
```

Comment:
what's the purpose of this cast?

Reply:
was to fix behaviour when I swapped an `any` to `string`, I think Laura fixed this so I can remove this